### DISCUSSION TITLE
it's still very slow to download hive/spark/flink when mirror-cdn enabled

### DISCUSSION THREAD

**Discussion by gabry-lab** [apache/kyuubi#2589] [Q&A] [answered]
The following command is still very slow to download hive/spark/flink even using https://dlcdn.apache.org as a CDN
`build/mvn clean package -Pmirror-cdn`
How to resolve it?

**Comment by gabry-lab** [comment, marked as answer]
Following command is very fast for China users
`build/mvn clean install -Dapache.archive.dist=https://mirrors.tuna.tsinghua.edu.cn/apache/`

**Reply by pan3793** [reply]
Yea, overwrite `apache.archive.dist` directly is an option if apache cdn mirror is slow
